classes/template/PKPTemplateManager.inc.php

```diff
@@ -284,7 +285,7 @@ public function initialize($request)
         }
 
         // Register custom functions
-        $this->registerPlugin('modifier', 'translate', '\__');
+        $this->registerPlugin('modifier', 'translate', [$this, 'smartyTranslateModifier']);
```

Comment:
TODO: Document the new syntaxes to use pluralized locale keys in the templates, I've extended the translation modifier and the translate tag.

Reply:
(this one also needs attention)

Reply:
I've documented the usage in the code, but perhaps it should be done in the PKP Docs as well (in case we have a section explaining the internal tags/filters).